src/app/code-set/code-set-details/code-set-details.component.ts

```diff
@@ -335,115 +276,61 @@ export class CodeSetDetailsComponent implements OnInit, OnDestroy {
     }
   };
 
-
-  askForPermanentDelete(): any {
-    if (!this.showPermDelete) {
-      return;
-    }
-
-    this.dialog
-      .openDoubleConfirmationAsync({
-        data: {
-          title: 'Delete permanently',
-          okBtnTitle: 'Yes, delete',
-          btnType: 'warn',
-          message: 'Are you sure you want to <span class=\'warning\'>permanently</span> delete this Code Set?'
+  save() {
+    if (this.validatorService.validateLabel(this.codeSetDetail.label)) {
```

Comment:
I'm wondering if validation could be done somehow on the `<mdm-inline-text-edit>` component instead, would maybe look nicer to have the validation/error message inline with the control as you change the form.